### PR TITLE
chore: update gcpchaos field description

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### CODE REVIEW

File: docs/simulate-gcp-chaos.md [hunk 1]
```diff
@@ -166,8 +166,8 @@ The following table shows the fields in the YAML configuration file.
 | mode | string | Indicates the mode of the experiment. The mode options include `one` (selecting a Pod at random), `all` (selecting all eligible Pods), `fixed` (selecting a specified number of eligible Pods), `fixed-percent` (selecting a specified percentage of the eligible Pods), and `random-max-percent` (selecting the maximum percentage of the eligible Pods). | None | Yes | `one` |
 | value | string | Provides parameters for the `mode` configuration, depending on `mode`. For example, when `mode` is set to `fixed-percent`, `value` specifies the percentage of pods. | None | No | 1 |
 | secretName | string | Indicates the name of the Kubernetes secret that stores the GCP authentication information. | None | No | cloud-key-secret |
-| project | string | Indicates the ID of GCP project. | None | Yes | your-project-id |
+| project | string | Indicates the name of GCP project. | None | Yes | real-testing-project |
```
Comment (g1eny0ung) [inline docs/simulate-gcp-chaos.md:169]: It looks like since the #223 update, this should remain as `ID`?

Comment (fewdan) [reply docs/simulate-gcp-chaos.md:169]: oh, i will modify it.

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Pull request merged**